packages/fxa-react/package.json

```diff
@@ -27,7 +27,7 @@
     "start": "pm2 start pm2.config.js",
     "stop": "pm2 stop pm2.config.js",
     "storybook": "npm run build-postcss && start-storybook -p 6007",
-    "test": "yarn clean && jest"
+    "test": "yarn clean && jest --env=jsdom-fourteen"
```

Comment:
Also, still trying to completely understand why this option is needed. These tests work under fxa-payments-server, where they originally lived in my branch. They fail without this option in fxa-react with an error about `MutationObserver`

More context from where I found the option mentioned: https://github.com/facebook/create-react-app/issues/7491#issuecomment-520157683

Reply:
(Aaand I just noticed that this very thing failed in CI when it worked locally. Ugh.)